_src/_assets/css/_states.scss

```diff
@@ -11,7 +12,8 @@
 }
 
 .states-list {
-  max-width: 60em;
+  max-width: 60rem;
+
   @media (min-width: $viewport-md) {
     display: grid;
     grid-template-columns: repeat(auto-fit, minmax(26em, 1fr));
```

Comment:
It looks like we might’ve lost the two-column layout on the states page. (Compare with [what’s currently live](https://covidtracking.com/data/).) Was that intentional? If not, I think bringing this down to ~20rem would restore it.